lib/resources/aws/aws_iam_policy.rb

```diff
@@ -112,15 +113,25 @@ def has_statement?(provided_criteria = {})
   def has_statement__validate_criteria(raw_criteria)
     recognized_criteria = {}
     EXPECTED_CRITERIA.each do |expected_criterion|
-      if raw_criteria.key?(expected_criterion)
-        recognized_criteria[expected_criterion] = raw_criteria.delete(expected_criterion)
+      [expected_criterion, expected_criterion.downcase, expected_criterion.to_sym, expected_criterion.downcase.to_sym].each do |variant|
```

Comment:
Could we split these over multiple lines like you have on https://github.com/inspec/inspec/pull/3129/files#diff-83a279cf431ca0d43273633052ea67bdR126